### PR TITLE
Disclaimers added and blur removed

### PR DESCRIPTION
**Pull request by richardverheyen**
So there have been some content changes to all the process sections and Disclaimers added to the end of the process sections. I had to add in a new div in the 'services' page, but it all works pretty good I think. Also Hannah said they asked for the blur to be removed from the 6 step process. 
Also I changed the arrangement of some of the margins so that the page has a bit less negative space.

### CODE REVIEW

File: src/styles/_about.scss [hunk 1]
```diff
@@ -77,7 +77,7 @@
     #process {
       background: $finsec-royal-blue;
       > div {
-        padding: 150px 20px;
+        padding: 150px 20px 100px;
```
Comment (janwerkhoven) [inline src/styles/_about.scss:80]: Have you tested this in mobile views as well?

File: src/styles/_services.scss [hunk 2]
```diff
@@ -102,6 +108,17 @@
     }
     #below-45 {
       background: $finsec-kashmir-blue;
+      > div {
+        > div {
+          padding-bottom: 6vw !important;
```
Comment (janwerkhoven) [inline src/styles/_services.scss]: Code smell, why do you need this?

File: src/styles/_about.scss (hunk 1 above)
Comment (richardverheyen) [reply src/styles/_about.scss:80]: Yes, it seems to work fairly well - spacing the <p class="disclaimer"> to a healthy distance inside the section

File: src/styles/_services.scss (hunk 2 above)
Comment (richardverheyen) [reply src/styles/_services.scss]: admittedly, laziness.
I'll fix it

File: src/templates/layout.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE HTML>
+<html lang="en">
+
+  <!-- <<<<<<< HEAD  -->
```
Comment (janwerkhoven) [inline src/templates/layout.html]: This is a left over of a bad merge. This is not supposed to be there. These can often break your code. Remove this please :)

File: src/templates/layout.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    <main>
+      {{#block "main"}}{{/block}}
+    </main>
+    {{> footer}}
+
+    <div id="outdated"></div>
+    <script src="/assets/js/outdated-browser.js" type="text/javascript"></script>
+
+  </body>
+  <!-- =======
```
Comment (janwerkhoven) [inline src/templates/layout.html]: Dude, this looks bad. There are 2 <heads> here Can you fix this.

Comment (richardverheyen) [reply src/templates/layout.html]: I'm fixing these as you comment, I removed it - I didn't want to delete too much until you checked it

**Review by janwerkhoven — CHANGES_REQUESTED** [review]
There are remains of a bad merge in your commit. Search for <<<<< and resolve please.

File: src/templates/layout.html (hunk 4 above)
Comment (janwerkhoven) [reply src/templates/layout.html]: If I were you, I would reset this one file. This is to important. You can pull in only this one file from `staging` by bashing `git checkout staging src/templates/layout.html` in your repo root. Then add, commit en push.

File: src/styles/_services.scss (hunk 2 above)
Comment (janwerkhoven) [reply src/styles/_services.scss]: It's ok to us `!important` at the end of a hierarchy. Or when mobile, desktop and inline jQuery styles need to overwrite each other quite dramatically. I don't find it a drama, but many other devs do...

File: src/styles/_about.scss (hunk 1 above)
Comment (janwerkhoven) [reply src/styles/_about.scss:80]: 👍

File: src/styles/_services.scss [hunk 5]
```diff
@@ -16,6 +16,9 @@
           flex-direction: row;
           align-items: center;
           margin: 0 auto;
+          &:nth-last-child(2) {
```
Comment (janwerkhoven) [inline src/styles/_services.scss:19]: 👌

**Review by janwerkhoven — APPROVED** [review]

**Comment by janwerkhoven** [comment]
Shipping ⛵️ , good job

**Pull request merged**